test/data/types-test/data.csv

```diff
@@ -0,0 +1,5 @@
+string,number,integer,boolean,date
```

Comment:
You want a lot more variety on booleans. You want to test all of the potential values in the spec e.g.

`true, True, TRUE, TrUe, 1 ...`
`false, False, FALSE, false, 0 ...`

Similarly for any other values.